extensions/bitwarden/src/search.tsx

```diff
@@ -86,11 +124,12 @@ function ItemList(props: {
               return a.favorite ? -1 : 1;
             })
             .map((item) => (
-              <ItemListItem
+              <BitwardenItem
                 key={item.id}
                 item={item}
+                lockVault={session.deleteToken}
                 refreshItems={refreshItems}
-                sessionToken={sessionToken}
+                sessionToken={session.token}
```

Comment:
Could we avoid passing two things down (`sessionToken` and `copyTotp`) to the `BitwardenItem` by wrapping the function definition in `useCallback`?

```typescript
  const copyTotp = useCallback(
    async (id: string) => {
      if (session.token) {
        const totp = await bitwardenApi.getTotp(id, session.token);
        Clipboard.copy(totp);
        exit();
      } else {
        showToast(Toast.Style.Failure, "Failed to fetch TOTP.");
      }
    },
    [session.token]
  );
```

We could also add a loading Toast while the command is running.

Reply:
Why is a useCallback necessary here ?

Reply:
You're right, there's no need for useCallback here - sorry!